src/pages/service/OurEngagement.jsx

```diff
@@ -0,0 +1,37 @@
+const OurEngagement = () => {
+    return (
+        <div className="py-6 px-6 md:px-16 lg:px-20 xl:px-28 2xl:px-36">
+            <div className="lg:text-4xl text-3xl text-dark-gray font-poppins">
+                <h1>Our Engagement</h1>
+                <h1>Modals</h1>
+            </div>
+            <div className="lg:flex md:flex justify-center items-center gap-8 pt-10 pb-20">
+                <div>
+                    <div className="bg-fade-white lg:h-96 h-80 w-full rounded-lg mb-4 lg:px-16 px-4 lg:pt-36 md:pt-24 pt-20">
+                        <div className="text-fade-black font-poppins">
+                            <h1 className="font-bold text-3xl">Monthly</h1>
+                            <h1 className="font-bold text-3xl">Retainer</h1>
+                        </div>
+                        <p className="pt-6 text-header-gray font-roboto opacity-80 lg:pr-12 pr-0 ">
+                            Yume Labs is a global UI UX Design and Industrial Design agency that helps your business
+                            scale through fail-proof design innovation systems.
+                        </p>
+                    </div>
+                </div>
+                <div>
```

Comment:
this is same as above code only text is changing can't we create a component for this in this file itself?